Sources/NIO/ByteBuffer-views.swift

```diff
@@ -187,3 +187,17 @@ extension ByteBuffer {
         self = view._buffer.getSlice(at: view.startIndex, length: view.count)!
     }
 }
+
+extension ByteBufferView: Equatable {
```

Comment:
Let's add `Hashable` as well: it's rare to have something that can usefully be `Equatable` but not `Hashable`.

Reply:
+1